test/ponse.js

```diff
@@ -112,3 +112,28 @@ test('ponse: send', (t) => {
     t.end();
 });
 
+test('ponse: get: content-type', async (t) => {
+    const filesToMimes = {
+        "320px-Floppy_disk_2009_G1" : "image/jpeg",
+        "320px-Floppy_disk_2009_G1.jpg" : "image/jpeg",
+        "294px-Railroad1860.png" : "image/png",
+        "294px-Railroad1860" : "image/png",
+        "hello.txt" : "text/plain; charset=UTF-8",
+    }
```

Comment:
Why some much files :)? I think 3: `jpeg`, `png` and `txt` will be enough :)

Reply:
Part of the motivation for this change was to ensure the `Content-Type` was correctly determined and conveyed for files that didn't have an extension.

I'm fine with deleting the `.{jpg,png}` files and just performing the tests on the extension-less files.